core/src/main/java/com/scalar/db/storage/dynamo/DynamoAdmin.java

```diff
@@ -1219,6 +1221,15 @@ public void importTable(String namespace, String table) {
         "import-related functionality is not supported in DynamoDB");
   }
 
+  @Override
+  public DistributedStorageMetadata getDistributedStorageMetadata(String namespace) {
+    return DistributedStorageMetadata.newBuilder()
+        .type("dynamo")
+        .name("dynamo")
+        .atomicityLevel(DistributedStorageAtomicityLevel.STORAGE)
```

Comment:
Ah, you meant `storage-level atomicity` is a capability of spanning multiple storages atomically?

I first thought if a storage guarantees `storage-level atomicity`, then users can update the whole storage atomically.
Maybe, it's misleading. It would be great if we could discuss it.

Reply:
I meant `storage-level atomicity`. In other words, with the DynamoDB storage adapter, we can execute mutations for tables in DynamoDB atomically. Anyway, let's discuss it later. Thanks.